### DISCUSSION TITLE
I have unread notifications which cannot be read 😢

### DISCUSSION THREAD

**Discussion by Rohlik** [community/community#111195] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hey guys 👋🏼,
it's being a few days what I have this notification on my right up corner:
![image](https://github.com/community/community/assets/2051154/156d27a6-4191-4bf0-9e6d-8c35a12ebf1f)
But I am unable to find that specific unread notifications, thus I am not even able to read it and clean this "✉️ with dot" 😞.

Here is what I see when I click on that icon:
![image](https://github.com/community/community/assets/2051154/cf0293a6-f027-47c6-842e-3cec6ec03842)

You can notice, that on left side and in the footer, there is "1" which indicate one unread notification, but notification itself is missing 🧐.

This is my first issue with this notification's system, so far it works nicely.
I tried to use procedure mentioned here (API call) https://docs.github.com/en/rest/activity/notifications?apiVersion=2022-11-28#mark-notifications-as-read, but I am getting `205` response without any output.
Any tips or way how to fix it?

**Comment by git-sumana** [comment]
Click on the done symbol buddy!

**Comment by Yuripetusko** [comment]
Same here
<img width="2156" alt="Screenshot 2024-03-09 at 18 34 32" src="https://github.com/community/community/assets/666169/56680709-b2d5-438a-b882-fe8bc21728bc">

**Reply by Yuripetusko** [reply]
Oh actually mine is solved now. Had to click "remove is_read filter" 🤦

**Reply by Rohlik** [reply]
I probably found a root cause of my issue based on e-mail notification.
Someone tagged bunch of random people, see figure:
![image](https://github.com/community/community/assets/2051154/d38f28b1-3827-4565-8671-d68043e76bd0)
Seems like a spammy action to me 🧐.
Sadly, that related gist was deleted or mark as private, because I can't open it - getting 404 (https://gist.github.com/no198/0e7b12f252aa5247420c56fb53061fec#gistcomment-4968341).
And that could be the reson for this race condition, when I can't see notification if the root object (gist) is unavailable 🤔.

**Reply by KingMario** [reply]
The same e-mail received

How do you fix it @Rohlik, please?

**Reply by alexisfontaine** [reply, marked as answer]
I also received this email a week ago. Found this [other discussion](https://github.com/orgs/community/discussions/111283) suggesting to use the notifications API directly:
```shell
gh api notifications -X PUT -F last_read_at=2024-03-12T00:00:00Z
```

**Reply by Rohlik** [reply]
Thank you @alexisfontaine, that works for me 🎉!

**Reply by myfr3nds** [reply]
T Thank you for being able to join this application

**Reply by am007ne0** [reply]
> I also received this email a week ago. Found this [other discussion](https://github.com/orgs/community/discussions/111283) suggesting to use the notifications API directly:
> 
> ```shell
> gh api notifications -X PUT -F last_read_at=2024-03-12T00:00:00Z
> ```

gh api notifications -X PUT -F last_read_at=2024-03-12T00:00:00Z

-

**Comment by doublekik** [comment]
![Screenshot_20241008-125109](https://github.com/user-attachments/assets/c729b4d4-f321-4aa9-a10f-e20ea7c2b2a9)
@Yuripetusko #__

**Reply by david1234121** [reply]
lol

**Reply by david1234121** [reply]
lol

**Reply by david1234121** [reply]
> lol

**Comment by arfat721** [comment]
hello.please notice that i am 50 years old and need more time for learning ofcource reading and practice more times..

**Reply by arfat721** [reply]
please send commons 3 times or more that i can remember those..thanks...

**Comment by fatimah7860** [comment]
Hi I try to type in my custom domain but it says someone else owns it when I have the legit domain. If anyone can help me and let me know how to verify my domain. That would be great m!

**Comment by Ultracenter1527** [comment]
![image](https://github.com/user-attachments/assets/5dced97a-7138-40a8-a375-c3057d4338f5)
